applications/app/controllers/InteractiveController.scala

```diff
@@ -139,27 +127,57 @@ class InteractiveController(
     }
   }
 
+  def renderDCRAmp(item: ItemResponse)(implicit request: RequestHeader): Future[Result] = {
+    itemResponseToModel(Future.successful(item)) flatMap {
+      case Left((model, blocks)) => {
+        val pageType = PageType.apply(model, request, context)
+        remoteRenderer.getAMPInteractive(wsClient, model, blocks, pageType)
+      }
+      case Right(other) => Future.successful(RenderOtherStatus(other))
+    }
+  }
+
+  def itemResponseToInteractivePickerInputData(item: ItemResponse)(implicit
+      request: RequestHeader,
+  ): Option[(DateTime, List[Tag])] = {
+    for {
+      capiDatetime <- item.content.flatMap(_.webPublicationDate)
```

Comment:
What is `webPublicationDate` for preview (non launched) content? I think this will dictate whether preview works as we'd like it to?

Reply:
I suspect it's `None` in which case actually non-launched content will return `NotFound`?

Reply:
To be experimented with :)

Reply:
Given the latest commit, this will simply default to legacy rendering